### ISSUE TITLE
S3 toString must not throw an exception

### ISSUE THREAD

**Issue by bassrock** [aws/aws-sdk-php#498]
All of a sudden I am getting the following error with the latest SDK:

```
PHP Fatal error:  Method CFSimpleXML::__toString() must not throw an exception in /vagrant/vendor/amazonwebservices/aws-sdk-for-php/services/s3.class.php on line 0


                                                                       
  [Symfony\Component\Debug\Exception\FatalErrorException]              
  Error: Method CFSimpleXML::__toString() must not throw an exception  
```

**Comment by bassrock** [comment]
I am using 1.6.2. Possibly I need to update

**Comment by jeremeamia** [comment]
This repo is for Version 2+ of the SDK. The repo for version 1 is at [amazonwebservices/aws-sdk-for-php](https://github.com/amazonwebservices/aws-sdk-for-php), but we no longer update that version of the SDK.

It is strange that you would get that error all of a sudden though, because we have not changed anything in the library in a long time. The only thing we changed recently in version 1 is that we tagged a 1.6.3 release that removed a particular CURLOPT that doesn't work on PHP 5.6. That doesn't seem related to your problem though.
